deb-get

```diff
@@ -2413,6 +2413,16 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_beyond-compare() {
+    if [ "${ACTION}" != "prettylist" ]; then
+        VERSION_PUBLISHED=$(curl -s -S https://www.scootersoftware.com/download.php | grep -Eo "Current Version:.*, released" | cut --complement -c -23 | sed 's/\,\ build\ /\./;s/\\, released//')
+        URL="https://www.scootersoftware.com/bcompare-${VERSION_PUBLISHED}amd64.deb"
```

Comment:
Multiple files are provided, depending on the architecture. The correct file should be selected based on `HOST_ARCH`, and the supported architectures should be listed in `ARCHS_SUPPORTED`. Read [CONTRIBUTING#creating-the-function](https://github.com/wimpysworld/deb-get/blob/main/CONTRIBUTING.md#creating-the-function) for further information.